renderer/src/lib/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { BigNumber, FilecoinNumber } from '@glif/filecoin-number'
+import { browseTransactionTracker } from './station-config'
+
+export function truncateString (value: string, size = 6) {
+  return `${value.slice(0, size)}...${value.slice(-size)}`
+}
+
+export function formatFilValue (value = '') {
+  return new FilecoinNumber(String(value), 'fil')
+    .decimalPlaces(3, BigNumber.ROUND_DOWN)
+    .toString()
+}
+
+export function openExplorerLink (hash: string) {
```

Comment:
It's not clear why this function needs to be curried, could you please remove the `return () =>` and use an arrow function in `WalletWidget` instead?

Reply:
It was to avoid having to use arrow functions every time it's used, but if it's not clear I'll remove it.